<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PtzControlPanel</class>
 <widget class="QWidget" name="PtzControlPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget_panel" native="true">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>221</width>
     <height>481</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton
{
	outline: 0px;
	border: 0px;
	border-radius: 3px;
	padding: 5px;
	color: #FFFFFF;
	background-color: #5F5F5F;
}
QToolButton:disabled
{
    background-color: rgb(190, 190, 190);
}
QToolButton:hover
{
	background-color: #2B9CC6;
}
QToolButton:pressed
{
	background-color: #5F5F5F;
}
QToolButton:checked
{
	background-color: #2B9CC6;
}
QLabel
{
    background-color: transparent;
    color: #4A4A4A;
    padding-left: 0px;
    padding-right: 0px;
}</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="horizontalSpacing">
     <number>25</number>
    </property>
    <property name="verticalSpacing">
     <number>15</number>
    </property>
    <item row="4" column="0" colspan="3">
     <widget class="QWidget" name="widget_zoom" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_zoom">
         <property name="minimumSize">
          <size>
           <width>37</width>
           <height>37</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../../resource/resource.qrc">:/ptz/ptz/zoom_b.png</pixmap>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="BaseSlider" name="slider_zoom">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLabel" name="label_iris">
      <property name="minimumSize">
       <size>
        <width>37</width>
        <height>37</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../../resource/resource.qrc">:/ptz/ptz/iris_b.png</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QToolButton" name="toolButton_RIGHT_UP">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../../resource/resource.qrc">
        <normaloff>:/ptz/ptz/rightup.png</normaloff>:/ptz/ptz/rightup.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>28</width>
        <height>28</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QToolButton" name="toolButton_autoScan">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../../resource/resource.qrc">
        <normaloff>:/ptz/ptz/center.png</normaloff>:/ptz/ptz/center.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>28</width>
        <height>28</height>
       </size>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QToolButton" name="toolButton_RIGHT_DOWN">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../../resource/resource.qrc">
        <normaloff>:/ptz/ptz/rightdown.png</normaloff>:/ptz/ptz/rightdown.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>28</width>
        <height>28</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QToolButton" name="toolButton_focus_add">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../../resource/resource.qrc">
        <normaloff>:/ptz/ptz/plus.png</normaloff>:/ptz/ptz/plus.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>28</width>
        <height>28</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QToolButton" name="toolButton_zoom_sub">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../../resource/resource.qrc">
        <normaloff>:/ptz/ptz/minus.png</normaloff>:/ptz/ptz/minus.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>28</width>
        <height>28</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="label_zoom_2">
      <property name="minimumSize">
       <size>
        <width>37</width>
        <height>37</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../../resource/resource.qrc">:/ptz/ptz/zoom_b.png</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="3">
     <widget class="QWidget" name="widget_speed" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_speed">
         <property name="minimumSize">
          <size>
           <width>37</width>
           <height>37</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../../resource/resource.qrc">:/ptz/ptz/speed_b.png</pixmap>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="BaseSlider" name="slider_speed">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QToolButton" name="toolButton_LEFT">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../../resource/resource.qrc">
        <normaloff>:/ptz/ptz/left.png</normaloff>:/ptz/ptz/left.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>28</width>
        <height>28</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QToolButton" name="toolButton_iris_add">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../../resource/resource.qrc">
        <normaloff>:/ptz/ptz/plus.png</normaloff>:/ptz/ptz/plus.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>28</width>
        <height>28</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QToolButton" name="toolButton_LEFT_DOWN">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../../resource/resource.qrc">
        <normaloff>:/ptz/ptz/leftdown.png</normaloff>:/ptz/ptz/leftdown.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>28</width>
        <height>28</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QToolButton" name="toolButton_zoom_add">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../../resource/resource.qrc">
        <normaloff>:/ptz/ptz/plus.png</normaloff>:/ptz/ptz/plus.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>28</width>
        <height>28</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QToolButton" name="toolButton_RIGHT">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../../resource/resource.qrc">
        <normaloff>:/ptz/ptz/right.png</normaloff>:/ptz/ptz/right.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>28</width>
        <height>28</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QToolButton" name="toolButton_iris_sub">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../../resource/resource.qrc">
        <normaloff>:/ptz/ptz/minus.png</normaloff>:/ptz/ptz/minus.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>28</width>
        <height>28</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="label_focus">
      <property name="minimumSize">
       <size>
        <width>37</width>
        <height>37</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../../resource/resource.qrc">:/ptz/ptz/focus_b.png</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QToolButton" name="toolButton_focus_sub">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../../resource/resource.qrc">
        <normaloff>:/ptz/ptz/minus.png</normaloff>:/ptz/ptz/minus.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>28</width>
        <height>28</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QToolButton" name="toolButton_UP">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../../resource/resource.qrc">
        <normaloff>:/ptz/ptz/up.png</normaloff>:/ptz/ptz/up.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>28</width>
        <height>28</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QToolButton" name="toolButton_DOWN">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../../resource/resource.qrc">
        <normaloff>:/ptz/ptz/down.png</normaloff>:/ptz/ptz/down.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>28</width>
        <height>28</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QToolButton" name="toolButton_LEFT_UP">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../../resource/resource.qrc">
        <normaloff>:/ptz/ptz/leftup.png</normaloff>:/ptz/ptz/leftup.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>28</width>
        <height>28</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="MaskPreset" name="widget_preset" native="true">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>80</y>
     <width>211</width>
     <height>461</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BaseSlider</class>
   <extends>QSlider</extends>
   <header>baseslider.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MaskPreset</class>
   <extends>QWidget</extends>
   <header>maskpreset.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resource/resource.qrc"/>
 </resources>
 <connections/>
</ui>
